index.yaml

```diff
@@ -3413,6 +3414,24 @@ entries:
     urls:
     - assets/rancher-operator/rancher-operator-0.1.400.tgz
     version: 0.1.400
+  - annotations:
+      catalog.cattle.io/auto-install: rancher-operator-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/experimental: "true"
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/namespace: rancher-operator-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/provides-gvr: clusters.rancher.cattle.io/v1
+      catalog.cattle.io/release-name: rancher-operator
+    apiVersion: v2
+    appVersion: 0.1.4
+    created: "2021-06-24T21:41:35.866839-07:00"
+    description: Control Rancher using GitOps
+    digest: 881ecadac8830da4ed862934a98ae047c957e4ccfb57ba0189a8dd759e81db68
+    name: rancher-operator
+    urls:
+    - assets/rancher-operator/rancher-operator-0.1.400+up0.1.4.tgz
+    version: 0.1.400+up0.1.4
```

Comment:
The two `0.1.400+up0.1.4`'s are still in the index

Reply:
This was what was generated by `make charts`. Manually removed it.